src/index.js

```diff
@@ -234,6 +234,8 @@ function validateLogIn() {
         loginArea.classList.add('hidden');
         requestsButton.classList.remove('hidden');
         viewTrips.classList.remove('hidden');
+    } else {
+        alert("Your username or password is incorrect")
```

Comment:
Wow. I can tell that you put a lot of effort into this. I am very impressed with how clean this was. And so easy? Good job.